nest-city-account/src/utils/subservices/line-logger.subservice.ts

```diff
@@ -26,73 +27,83 @@ export class LineLoggerSubservice implements LoggerService {
-      }
-      else {
-        logfmtStringItems = ` message="${escapeForLogfmt(stringItems)}"`
-      }
-    }
-
-    const logfmtOtherItems = otherItems.map((param) => ToLogfmt(param)).join(' ')
-
-    let context = ''
-    if (this.context) {
-      context = ` context="${this.context}"`
-    }
+    const stringMessages = completeArray
+      .filter((item): item is string => typeof item === 'string')
+      .join(' ')
+
+    const otherItems = completeArray.filter(
+      (item): item is string => typeof item !== 'string',
+    )
+
+    const formattedStringMessages = this.formatStringMessage(stringMessages)
+
+    const formattedOtherItems = otherItems
+      .map((item) => ToLogfmt(item))
+      .join(' ')
+
+    const formattedContext = this.context ? `context="${this.context}"` : ''
+
+    const colorStart = this.color ? colorCode : ''
+    const colorEnd = this.color ? '\u001B[0m' : ''
 
+    // eslint-disable-next-line no-console
     console.log(
-      `${this.color ? colorCode : ''}process="[Nest]" processPID="${
-        process.pid
-      }" datetime="${getCurrentDateTime()}" severity="${severity}"${context}${logfmtStringItems} ${logfmtOtherItems}${
-        this.color ? '\x1b[0m' : ''
-      }`
+      [
+        colorStart,
+        [
+          `process="[Nest]"`,
+          `processPID="${process.pid}"`,
+          `datetime="${getCurrentDateTime()}"`,
+          `severity="${severity}"`,
+          formattedContext,
+          formattedStringMessages,
+          formattedOtherItems,
+        ]
+          .filter(Boolean)
+          .join(' '),
+        colorEnd,
+      ]
+        .filter(Boolean)
+        .join(''),
     )
   }
 
-  log(message: unknown, ...optionalParams: unknown[]) {
-    this.printLog('LOG', message, optionalParams, '\x1b[32m')
+  log(message: unknown, ...optionalParams: unknown[]): void {
+    this.printLog('LOG', message, optionalParams, '\u001B[32m')
   }
 
-  fatal(message: unknown, ...optionalParams: unknown[]) {
-    this.printLog('FATAL', message, optionalParams, '\x1b[1m')
+  fatal(message: unknown, ...optionalParams: unknown[]): void {
+    this.printLog('FATAL', message, optionalParams, '\u001B[1m')
   }
 
-  error(message: unknown, ...optionalParams: unknown[]) {
-    this.printLog('ERROR', message, optionalParams, '\x1b[31m')
+  error(message: unknown, ...optionalParams: unknown[]): void {
+    this.printLog('ERROR', message, optionalParams, '\u001B[31m')
   }
 
-  warn(message: unknown, ...optionalParams: unknown[]) {
-    this.printLog('WARN', message, optionalParams, '\x1b[33m')
+  warn(message: unknown, ...optionalParams: unknown[]): void {
+    this.printLog('WARN', message, optionalParams, '\u001B[33m')
   }
 
-  debug?(message: unknown, ...optionalParams: unknown[]) {
-    this.printLog('DEBUG', message, optionalParams, '\x1b[35m')
+  debug(message: unknown, ...optionalParams: unknown[]): void {
+    this.printLog('DEBUG', message, optionalParams, '\u001B[35m')
```

Comment:
same as for the ANSI_RESET above, can be copy pasted (after edit) from here: [file-ansi-codes-color-and-others](https://gist.github.com/mgumiero9/665ab5f0e5e7e46cb049c1544a00e29f#file-ansi-codes-color-and-others)